### DISCUSSION TITLE
Why the status of pull request show merged?(there is actually no merge operation)

### DISCUSSION THREAD

**Discussion by whtiehack** [community/community#29239] [Pull Requests] [answered]
https://github.com/node-pinus/pinus/pull/531

I'm confused.

**Comment by ChaituVR** [comment]
Same here https://github.com/snapshot-labs/snapshot-strategies/pull/765

**Comment by queenofcorgis** [comment, marked as answer]
👋🏼 @whtiehack @ChaituVR 

Would you mind elaborating please, giving a few more details might help someone give you a nudge in the right direction.

**Reply by mktcode** [reply]
There are PRs being shown as merged, even though no admin merged them. I assume it has something to do with force-pushing back to the current upstream commit. Somthing like that. I tried to reproduce that with my own repos but they were shown as closed then. I didn't merge nor close [this PR](https://github.com/mktcode/test/pull/9) but by force-pushing to the PR branch you seem to be able to change the PR state, which is what's happening here, I assume.

**Reply by queenofcorgis** [reply]
This is likely a result of the pull request being merged indirectly, this can happen in a number of ways, for example, changes to be introduced by that pull request were added into the BASE branch outside of that pull request (this is typically through a direct push or via another pull request) and the status of the pull request was updated to reflect that. I understand that this is confusing so I've taken both of your feedback and passed it along in an internal Issue to our team so we can enhance this experience to be more clear.

**Reply by ChaituVR** [reply]
@queenofcorgis i can confirm that in my case same changes were not merged into BASE branch by direct push or PR . Just when user closed the PR, i saw 'merged' status, so this could be something else. But not really sure how to reproduce
